resources/views/backend/layouts/login.blade.php

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html> 
+<html lang="en"> 
+<head> 
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> 
+  <!-- Meta, title, CSS, favicons, etc. --> 
+  <meta charset="utf-8"> 
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"> 
+  <meta name="viewport" content="width=device-width, initial-scale=1"> 
+
+  <title>{!! trans('labels.promotion_login') !!}</title> 
+
+  <!-- CSS --> 
+  <link href="{{ asset('backend/css/vendor.css') }}" rel="stylesheet"> 
+
+  <!-- Font Awesome --> 
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css"> 
+
+  <!-- Ionicons --> 
+  <link rel="stylesheet" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css"> 
+
+</head> 
+
+<body class="login"> 
+  <div> 
+    <div class="login_wrapper"> 
+      <div class="animate form login_form"> 
+        <section class="login_content"> 
+        <form method="POST" role="form" id="form-login" action="{{ url('admin\login') }}">
+        {{ csrf_field() }}
+
+        <h1>{!! trans('labels.login_form') !!}</h1> 
+            
+        {{-- Email --}}
```

Comment:
I think if you want to comment, please use the html comment, don't use blade in this case